Utils/GameUtils.cs

```diff
@@ -114,6 +114,12 @@ public static bool IsPMC(this IPlayer player)
             return player.Profile.Side == EPlayerSide.Bear || player.Profile.Side == EPlayerSide.Usec;
         }
 
+        public static bool IsDedicatedServer(this IPlayer player)
+        {
+            string pattern = @"^dedicated_[a-fA-F0-9]{24}$";
+            return Regex.IsMatch(player.Profile.GetCorrectedNickname(), pattern);
```

Comment:
On second through I feel like this might need to be `player.Profile.Info.Nickname` instead as I am assuming `GetCorrectedNickname()` will return a pretty name for scavs? Tbh I haven't tested this, I just noticed this issue in game.